exporter/ethstore.go

```diff
@@ -48,11 +48,22 @@ func StartEthStoreExporter(bnAddress string, enAddress string, updateInterval, e
 		ese.Sleep = time.Minute
 	}
 
+	// Reexport days if specified
+	if startDayReexport != -1 && endDayReexport != -1 {
+		for day := startDayReexport; day <= endDayReexport; day++ {
+			err := ese.exportDay(strconv.FormatInt(day, 10), true)
+			if err != nil {
+				logger.WithError(err).Errorf("error reexporting eth.store day %d in database", day)
```

Comment:
Personally I would use this opportunity to switch to `utils.LogError` whenever I have to add a new error log in an environment that does not use `utils.LogError`. Even for lines not touched by your PR like https://github.com/gobitfly/eth2-beaconchain-explorer/blob/4ad8da36a4a34dfa029705fc50f96a971f7dd85e/exporter/ethstore.go#L209